tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -99,7 +99,7 @@ Connect containers to multiple networks non-overlapping
 
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --net cross2-network2 --name cross2-container2 debian ping -c2 ${ip}
     Should Not Be Equal As Integers  ${rc}  0
-    Should contain  ${output}  Destination Port Unreachable
+    Should Contain  ${output}  2 packets transmitted, 0 packets received, 100% packet loss
```

Comment:
Very happy with this test of isolation - can we also have a test that ensures we get the same response when trying to connect to management-network assets - that was the original issue and I'm not seeing a test that checks that isolation.

IIRC @mreferre also had a request that containers be able to communicate with one another via the port fowarding on the public interface. While it's a nasty indirect path, that is the public network access so if someone passes it to a container it should be accessible. At least we should have a test that asserts current behaviour and doc it appropriately.